README.md

```diff
@@ -187,24 +189,29 @@ function onXRFrame(time, xrFrame) {
 // Layer interface
 //
 
-interface XRLayer {
-  XRSubImage? getViewSubImage(XRView view);
+interface XRSubImage {
+  readonly attribute XRViewport viewport;
+}
+
+interface XRWebGLFramebufferSubImage : XRSubImage {
+  readonly attribute WebGLFramebuffer framebuffer;
+}
 
+interface XRWebGLTextureSubImage : XRSubImage {
+  readonly attribute unsigned long imageIndex;
+  readonly attribute WebGLTexture colorTexture;
+  readonly attribute WebGLTexture depthTexture;
```

Comment:
This should be nullable and the name should more accurately be `depthStencilTexture`, since if a stencil channel is requested it will have to be exposed here.